### DISCUSSION TITLE
2006/03/07/gloria%E5%86%99%E7%9A%84%E4%B8%80%E7%AF%87%E5%B0%8F%E6%96%87

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#77] [Announcements]
# 2006/03/07/gloria%E5%86%99%E7%9A%84%E4%B8%80%E7%AF%87%E5%B0%8F%E6%96%87

今天跟朋友去吃素了，慕名找到了“早吃素”的枣子树。刚拐到嵩山路上就把吃素的心情和意境破坏了。周围乱糟糟闹哄哄的环境真的跟网上介绍的格调大不相符，虽然枣子树自己门前的布局还算雅致。排了一会儿队终于坐进了餐馆里，不知道是因为期望过高呢还是怎么的，就是感觉不太好。店里面黑黑的，吵吵的，找不到一点吃素的感觉；尤其是还遇到了一个有点智障的服务员，光是点菜就折腾了我们半天。不是我故意找茬啊，且不说写字跟不上吧，明明是要无锡酱排，她偏偏说是黑椒醋排，而且前前后后重复了三遍；好不容易点完菜了，刚想清静一会儿，谁知道那服务生扭过头又来问，“要点菜吗？”旁边有个知趣的服务员朝我们略带歉意的无可奈何的笑笑，算是给我们的安慰吧。

https://blog.qizhen.xyz/2006/03/07/gloria%E5%86%99%E7%9A%84%E4%B8%80%E7%AF%87%E5%B0%8F%E6%96%87

**Comment by ruanqizhen** [comment]
之前博客下的留言：

ERICA
2006年05月7日 23:15 编辑
一家可以吃到你低血糖的餐厅

回复
Tian
2006年03月16日 20:34 编辑
good good 叫Gloria也开个博吧？

回复
yoyo
2006年03月12日 19:12 编辑
油是有点多，味道还能接受
